unixfs/src/pb/merkledag.rs

```diff
@@ -0,0 +1,102 @@
+// Modified automatically generated rust module for 'merkledag.proto' file
+#![allow(rust_2018_idioms)]
+#![allow(non_snake_case)]
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(unused_imports)]
+#![allow(unknown_lints)]
+#![allow(clippy::all)]
+#![cfg_attr(rustfmt, rustfmt_skip)]
+use super::*;
+use quick_protobuf::sizeofs::*;
+use quick_protobuf::{BytesReader, MessageRead, MessageWrite, Result, Writer, WriterBackend};
+use std::borrow::Cow;
+use std::convert::TryFrom;
+use std::io::Write;
+use std::ops::Deref;
+use std::ops::DerefMut;
+#[derive(Debug, Default, PartialEq, Clone)]
+pub struct PBLink<'a> {
+    pub Hash: Option<Cow<'a, [u8]>>,
+    pub Name: Option<Cow<'a, str>>,
+    pub Tsize: Option<u64>,
+}
+impl<'a> MessageRead<'a> for PBLink<'a> {
```

Comment:
fmt: a line break would be nice here

Reply:
While rustfmt did change this file on updating from 1.43 to 1.44 it doesn't want to add those empties with the rustfmt_skip present.